vendor/assets/javascripts/selectize.js

```diff
@@ -2652,7 +2652,8 @@
 			var invalid, self = this;
 			if (self.isRequired) {
 				if (self.items.length) self.isInvalid = false;
-				self.$control_input.prop('required', invalid);
+				self.$control_input.prop('required', self.$input.val() == '');
+				self.$input.prop('required', false);
```

Comment:
Oh sorry, just read the rest of the PR. Please add a comment at the top with the link to that comment!